hw5/WordsGame.java

```diff
@@ -0,0 +1,64 @@
+package org.hillel.hw5;
+
+import java.util.Random;
+import java.util.Scanner;
+
+
+//First version fo Words Game
+public class WordsGame {
+
+    private final String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado" ,
+            "broccoli", "carrot", "cherry", "garlic", "grape", "melon", "leak",
+            "kiwi", "mango", "mushroom", "nut", "olive", " pea", "peanut", "pear",
+            "pepper", "pineapple", "pumpkin", "potato"};
+
+    int indexWord = new Random().nextInt(words.length);
```

Comment:
Would be better to make wordLength and indexWord private.
Move the initialization of these variables to the initialization block.